packages/open-payments/src/openapi/generated/auth-server-types.ts

```diff
@@ -164,14 +164,22 @@ export interface components {
      * limits-outgoing
      * @description Open Payments specific property that defines the limits under which outgoing payments can be created.
      */
-    "limits-outgoing": Partial<unknown> & {
+    "limits-outgoing": Partial<{
       receiver?: external["schemas.yaml"]["components"]["schemas"]["receiver"];
-      /** @description All amounts are maxima, i.e. multiple payments can be created under a grant as long as the total amounts of these payments do not exceed the maximum amount per interval as specified in the grant. */
-      debitAmount?: external["schemas.yaml"]["components"]["schemas"]["amount"];
-      /** @description All amounts are maxima, i.e. multiple payments can be created under a grant as long as the total amounts of these payments do not exceed the maximum amount per interval as specified in the grant. */
-      receiveAmount?: external["schemas.yaml"]["components"]["schemas"]["amount"];
       interval?: components["schemas"]["interval"];
-    };
+    }> &
+      Partial<{
+        receiver?: external["schemas.yaml"]["components"]["schemas"]["receiver"];
+        interval?: components["schemas"]["interval"];
+        /** @description All amounts are maxima, i.e. multiple payments can be created under a grant as long as the total amounts of these payments do not exceed the maximum amount per interval as specified in the grant. */
+        debitAmount: external["schemas.yaml"]["components"]["schemas"]["amount"];
+      }> &
+      Partial<{
+        receiver?: external["schemas.yaml"]["components"]["schemas"]["receiver"];
+        interval?: components["schemas"]["interval"];
+        /** @description All amounts are maxima, i.e. multiple payments can be created under a grant as long as the total amounts of these payments do not exceed the maximum amount per interval as specified in the grant. */
+        receiveAmount: external["schemas.yaml"]["components"]["schemas"]["amount"];
+      }>;
```

Comment:
These types would allow the client to take in both `receiveAmount` or `debitAmount` (because of the old openapi-typescript lib). I've updated it [here](https://github.com/interledger/open-payments/pull/567) (and upgraded to TS 5 [here](https://github.com/interledger/open-payments/pull/569)) so we can get those in first)